specification/resources/resource-manager/Microsoft.Management/2017-08-31-preview/management.json

```diff
@@ -0,0 +1,622 @@
+     "readOnly": true
+    },
+    "properties": {
+     "title": "Properties",
+     "x-ms-client-flatten" : true,
+     "$ref": "#/definitions/ManagementGroupPropertiesWithHierarchy"
+    }
+   }
+  },
+  "ManagementGroupPropertiesWithHierarchy": {
+   "description": "The generic properties of a management group.",
+   "type": "object",
+   "properties": {
+    "tenantId": {
+     "type": "string",
+     "format": "uuid",
+     "description": "The AAD Tenant ID associated with the management group.",
+     "example": "10000000-0000-0000-0000-000000000000"
+    },
+    "displayName": {
+     "type": "string",
+     "description": "The friendly name of the management group.",
+     "example": "Sample display name"
+    },
+    "details": {
+     "title" : "Details",
+     "$ref": "#/definitions/ManagementGroupDetailsProperties"
+    },
+    "children": {
+     "description": "The list of children.",
+     "type": "array",
+     "items": {
+      "$ref": "#/definitions/ManagementGroupRecursiveChildInfo"
+     }
+    }
+   }
+  },
+  "ManagementGroupRecursiveChildInfo": {
+   "description": "The unique identifier (ID) of a management group.",
+   "properties": {
+    "childType": {
+     "title" : "Managment Group Recursive Child Info",
+     "$ref": "#/definitions/ManagementGroupType"
+    },
+    "childId": {
+     "type": "string",
+     "description": "The ID of the child resource (management group or subscription).",
+     "example": "/providers/Microsoft.Management/managementGroups/40000000-0000-0000-0000-000000000000"
+    },
+    "displayName": {
+     "type": "string",
+     "description": "The friendly name of the child resource.",
+     "example": "Sample display name"
+    },
+    "tenantId": {
+     "type": "string",
+     "format": "uuid",
+     "description": "(Optional) The AAD Tenant ID associated with the child resource.",
+     "example": null
+    },
+    "children": {
+     "description": "The list of children.",
+     "type": "array",
+     "items": {
+      "$ref": "#/definitions/ManagementGroupRecursiveChildInfo"
+     }
+    }
+   }
+  }
+ },
+ "parameters": {
+  "GroupIdParameter": {
+   "name": "groupId",
+   "in": "path",
+   "required": true,
+   "type": "string",
+   "format": "uuid",
+   "description": "Management Group ID."
+  },
+  "ApiVersionParameter": {
+   "name": "api-version",
+   "in": "query",
+   "required": true,
+   "type": "string",
+   "description": "Version of the API to be used with the client request. The current version is 2017-08-31-preview."
+  },
+  "PageTokenParameter": {
+   "name": "token",
+   "in": "query",
+   "required": false,
+   "type": "string",
+   "description": "Page continuation token is only used if a previous operation returned a partial result. \nIf a previous response contains a nextLink element, the value of the nextLink element will include a token parameter that specifies a starting point to use for subsequent calls.\n"
```

Comment:
please specify `x-ms-parameter-location: method`, refer https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-parameter-location

Reply:
Is this a new requirement? None of our older RPs has this property defined.

Reply:
this is not a new requirement - if you don't apply this attribute then these become properties of the client (`ManagementGroups.cs`), only the global properties (e.g. apiVersion, subscriptionId) which are applicable across all operations are supposed to be in the client level.